masonry/src/contexts.rs

```diff
@@ -847,7 +870,7 @@ impl LayoutCtx<'_> {
         self.assert_layout_done(child, "place_child");
         if origin != self.get_child_state_mut(child).origin {
             self.get_child_state_mut(child).origin = origin;
-            self.get_child_state_mut(child).needs_window_origin = true;
+            self.get_child_state_mut(child).translation_changed = true;
```

Comment:
would `origin_changed` (or `window_origin_changed`) make more sense (as this is the defacto end-result if understand this correctly)?

Reply:
AIUI, this flag means that "my local translation has changed", and so the "global translation" needs to be updated.

Reply:
Yeah, `translation` and `window_origin` have different semantics.